roles/openshift_control_plane/tasks/upgrade_facts.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
This file is not used anymore in openshift_master; it can be removed from here as well.

I'll create a patch in master to remove this.